content/documentation/pages/7-resources/3-faq.md

```diff
@@ -415,3 +415,12 @@ task launch myBatchApp --arguments="team=cubs"
 However, the preferred way is to write your task or batch application such that it can handle being restarted with a new job instance. One way to do this is to set a `JobParamsIncrementer` for your batch job, as discussed in the Spring Batch [reference guide](https://docs.spring.io/spring-batch/trunk/reference/html/configureJob.html#JobParametersIncrementer).
 
 <!--END_QUESTION-->
+
+<!--QUESTION#useexistingbatchtables-->
+
+Can I use Spring Cloud Data Flow on my existing Spring Batch tables?
+
+No. Spring Cloud Data Flow creates its own schema including the Spring Batch tables.
+The Spring Batch tables created by Spring Cloud Data Flow will be used by your Spring Batch applications and require no change to your apps, they just need to use the same data source configuration as Spring Cloud Data Flow.
```

Comment:
The "used by your Spring Batch applications" is a bit unclear. I understand this is for the jobs launched by SCDF, but folks might get confused by referring this to the standalone jobs. Perhaps we could call out the explicit connect with SCDF.